### PR TITLE
Northstar authentication! :lock:

### PR DESCRIPTION
**Pull request by deadlybutter**
This pull request adds Northstar authentication to phoenix-next and makes the necessary config & db changes to make it work. I also got rid of a few files no longer needed.

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -2,28 +2,15 @@
...
-
-    /**
-     * The attributes that should be hidden for arrays.
-     *
-     * @var array
-     */
-    protected $hidden = [
-        'password', 'remember_token',
-    ];
+    protected $table = 'users';
```
Comment (DFurnes) [inline app/Models/User.php]: Is this necessary? Laravel should pluralize the classname by default.

File: config/auth.php [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 return [
 
+    'model' => App\Models\User::class,
```
Comment (DFurnes) [inline config/auth.php]: Can you add a comment specifying that this is a hack until DoSomething/gateway#65 is fixed?

Comment (deadlybutter) [inline config/auth.php]: :indeed:

File: app/Models/User.php (hunk 1 above)
Comment (deadlybutter) [inline app/Models/User.php]: Not necessary, had some generic bland SQL errors and was working through all possible things going wrong. I shall remove!

**Pull request merged**